### ISSUE TITLE
wiki: non-ascii characters in revision reason produce HTTP 400 error

### ISSUE THREAD

**Issue by grischard** [reddit/reddit#1393]
Summary: When submitting a revision for a wiki page, including a non-ascii character in the revision reason results in a 400 error. The error isn't shown on the webpage.
Steps to reproduce:
1. Edit any wiki page
2. Enter 'é' as your revision reason
3. Click on 'save page'

Expected result: revision is saved, user is redirected to the 'view' page for that wiki page.

Actual result: spinning wheel is shown for a brief moment. Nothing happens for the user. In the background, a 400 error is returned.

Workaround: submitting revision reasons in ascii avoids the issue. This might be hard or impossible in some languages.

**Comment by xiongchiamiov** [comment]
Yep, has to do with [this section](https://github.com/reddit/reddit/blob/efb9d0b35b08bf160571c686121b97a084f33215/r2/r2/lib/validator/validator.py#L598-L602) of code.
